File: 3746. Minimum String Length After Balanced Removals-1.cpp
class Solution {
public:
  int minLengthAfterRemovals(string s) {
    int cntA = 0, cntB = 0;
    for (auto ch : s)
      if (ch == 'a')
        cntA++;
      else
        cntB++;
    return abs(cntA - cntB);
  }
};

File: 3746. Minimum String Length After Balanced Removals-1.py
class Solution:
    def minLengthAfterRemovals(self, s: str) -> int:
        cntA = cntB = 0
        for x in s:
            if x == "a":
                cntA += 1
            else:
                cntB += 1
        return abs(cntA - cntB)
